_LowLevelCode/cpp/test/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+set(TESTS_BIN_DIR "${CMAKE_BINARY_DIR}/tests/bin")
+
+# On Windows we need to point the tests to the Matlab dlls. Define a
+# TEST_ENV_PATH variable that has the location prepended to the system path.
+# When adding tests, this can be set as the system path for those tests.
+if(WIN32)
+    file(TO_CMAKE_PATH "$ENV{PATH}" SYS_PATH)
+    set(TEST_ENV_PATH "${SYS_PATH};${Matlab_DLL_DIR}")
+    string(REPLACE ";" "\;" TEST_ENV_PATH "${TEST_ENV_PATH}")
```

Comment:
what's this `;` magic doing?

Reply:
`;` is a list separator in CMake, so to get the string, we need to escape it. I've added a comment explaining the need for the escape character.